.github/workflows/ci.yml

```diff
@@ -221,3 +221,12 @@ jobs:
           severity: warning
           ignore_paths: tests
           ignore_names: user_data.sh
+  url-suffix-check:
+    name: URL Suffix Check
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run URL Suffix Detection Script
+      run: |
+        chmod +x ./util/detect_bad_urls.sh
```

Comment:
No need for custom logic. See https://github.com/aws/aws-parallelcluster-cookbook/blob/develop/.github/workflows/unsafe-patterns-checker.yml

Reply:
Done